devtools/server/actors/replay/connection.js

```diff
@@ -542,7 +544,57 @@ class Recording extends EventEmitter {
   }
 }
 
-function canRecordUrl(url) {
+function checkShouldValidateUrl() {
+  if (gShouldValidateUrl !== null) {
+    return Promise.resolve(gShouldValidateUrl)
+  } else {
+    return fetch(getAPIServer(), {
```

Comment:
do we care about multiple in flight requests. Probably not...

Reply:
I don't think so. If either "fail" we want to handle